src/FileType_XML.f90

```diff
@@ -1308,11 +1309,27 @@ SUBROUTINE processTagAttributes(elStartTag,nattr,anames,avalues,ierr)
         ENDDO
       ENDIF
 
-      !Get the names (names precede the '=' character with no whitespace)
+      !Get the names (names precede the '=' character with possible whitespace preceeding '=')
       !attribute names cannot contain whitespace, must be unique and preceded
       !by whitespace
       DO i=1,nattr
-        namestt=SCAN(startTag(1:anchorLoc(i)),' '//CHAR(9)//CHAR(10)//CHAR(13),.TRUE.)
+        ! March backwards from anchorLoc(i) through char array if whitespace preceeds the '='
```

Comment:
This is to handle XML files from some FMU export utilities that produce XML files with misc. whitespace.  Ex FMU XML:

```
<?xml version="1.0" encoding='us-ascii'?>

<fmiModelDescription
  fmiVersion               = "2.0"
  modelName                = "Rectifier"
  guid                     = "{355fd1a1-5000-069a-f93c-629026df9008}"
  description              = "Model Rectifier"
  numberOfEventIndicators  = "6">

  <CoSimulation
    modelIdentifier                        = "Rectifier"
    canInterpolateInputs                   = "false">
    <SourceFiles>
        <File name = "Rectifier.c"/>
    </SourceFiles>
  </CoSimulation>

  <UnitDefinitions>
    <Unit name =  "V" />
    <Unit name =  "s" />
  </UnitDefinitions>

  <DefaultExperiment
    startTime =  "0.00000000000000000e+00"
    stopTime="1.00000000000000000e-01"
    stepSize="1.00000000000000000e-07"/>

  <ModelVariables>
    <ScalarVariable
      name           = "internalTime"
      valueReference = "0"
      description    = "internal time"
      causality      = "independent" >
      <Real 
        unit     = "s"/>
    </ScalarVariable>   <!--index="1" -->
    <ScalarVariable
      name           = "outputs"
      valueReference = "1"
      description    = "Rectifier1.Capacitor1.v"
      initial        = "exact"
      causality      = "output" >
      <Real 
        unit     = "V"
        start    = "2.82842712474619020e+02"/>
    </ScalarVariable>   <!--index="2" -->
  </ModelVariables>

</fmiModelDescription>

```

Reply:
Cool.  Good update.